content/building-blocks/automate-and-execute-your-work/reproducible-work/google_cloud_docker.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Import and Run a python environment on Google cloud with Docker" 
+description: "Learn how to import and run a python environment on google cloud with docker"
+keywords: "docker, images, container, environments, virtual, Python, Jupyter, Jupyter notebook, environment, google cloud, cloud"
+weight: 2
+author: Diego Sanchez Perez
+draft: true
+aliases:
+
+
+---
+
+{{% warning %}}
+
+## Some comments on the draft
+

```

Comment:
The original intention with this warning was to provide some context and express a few general ideas about the building block, posible future lines of expansion etc. It was not intended to be on display in the final version for TSH's readers but just for reviewers until the building block is more mature/finished.

In any case if any piece of information from here is finally deemed useful it can be keeped an integrated within the rest of the building block!